manifests/helm-watcher/helm_watcher_service_account.yaml.tpl

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: helm-watcher
+  namespace: {{ .Namespace }}
```

Comment:
do we need another service account? Can we not just bind to the existing one?

Reply:
yes, nope

Reply:
:trollface:

Reply:
wait what :joy:

Reply:
Yeah, we do, because the other service account is different with different roles attached. Also, I don't really want to bind to it, it belongs to the running ui thing. And I don't think everyone wants that running maybe, or I don't know. I was reluctant to re-use that.

Reply:
but its the same affect in the end? The UI, app API and profiles API all run in the same deployment. Adding the extra service account just adds more yaml, has no change on the outcome?

Reply:
Technically, yes. I could even modify the wego-app yamls themselves and add extra permissions there.

But it felt like a sin. :D I'm reluctant, because what if we decide that the helm-watcher should be in its own deployment since it's almost a separate entity, unlike the profiles service?

Reply:
> But it felt like a sin. :D I'm reluctant, because what if we decide that the helm-watcher should be in its own deployment since it's almost a separate entity, unlike the profiles service?

then we still have to update the resources of `wego-app` to remove stuff in that case as the Binding/Deployment-spec will change anyway

Reply:
Also! The helm-watcher needs cross namespace access. So I can't modify the existing role.

Reply:
yeah. I think just adding the `role` and `rolebinding` to `wego-app` is nicer than a new `helm-watcher` dir with `role`, `rolebinding` and `serviceaccount`. Also baring in mind that you still have to update the `deployment` in `wego-apps` to reference the `serviceaccount`, so it becomes overlapping already

Reply:
Discussion outcome: I'll pull these together into the `wego-app` folder so I don't have to have separate functions.